docs/user-guide/metadata.md

```diff
@@ -141,3 +143,24 @@ jobs:
 These settings will result in Git checks that look like:
 
 ![PR checks](./assets/pr-checks.png)
+
+### Coverage and Test Results
+
+You can populate coverage and test results on build page from a Screwdriver build through using metadata. Screwdriver UI will read from `tests.coverage` and `tests.results` in metadata and display them.
+
+Example screwdriver.yaml should look something like below:
+
+```
+jobs:
+  main:
+    steps:
+      - set-coverage-and-test-results: |
+          meta set tests.coverage  100 # this should be the coverage percentage number
+          meta set tests.results 10/10 # this should be `pass_tests_number/total_tests_number`
```

Comment:
can we take this as `tests.results.success` & `tests.results.total` so that we can verify this data as Number type ?

Reply:
what's wrong with with the regex `/^\d+\/\d+$/`?